lib/rust/ensogl/example/slider/src/lib.rs

```diff
@@ -50,16 +49,15 @@ pub fn main() {
     });
 }
 
-fn make_number_picker(app: &Application) -> Leak<selector::NumberPicker> {
-    let slider = app.new_view::<selector::NumberPicker>();
-    slider.frp.resize(Vector2(200.0, 50.0));
-    app.display.add_child(&slider);
-    Leak::new(slider)
-}
-
-fn make_range_picker(app: &Application) -> Leak<selector::NumberRangePicker> {
-    let slider = app.new_view::<selector::NumberRangePicker>();
-    slider.frp.resize(Vector2(400.0, 50.0));
+fn make_slider(app: &Application) -> Leak<slider::Slider> {
+    let slider = app.new_view::<slider::Slider>();
+    slider.frp.set_width(200.0);
+    slider.frp.set_height(50.0);
+    slider.frp.set_background_color(color::Lcha(0.8, 0.0, 0.0, 1.0));
+    slider.frp.set_max_value(5.0);
+    slider.frp.set_default_value(1.0);
+    slider.frp.set_value(1.0);
+    slider.frp.set_max_disp_decimal_places(8);
```

Comment:
If you don't set these values, the slider still shows up with some decent default, right?

Reply:
I had gone too far in removing defaults previously. Now a viable slider will be constructed without any additional initialization.